multipart.go

```diff
@@ -39,6 +39,9 @@ func MultipartUploadTest(store types.Storager, path string) {
 		log.Fatalf("CreateMultipart %v: %v", path, err)
 	}
 
+	// `isValidMultipartID` indicates whether the multipartId is valid.
+	var isValidMultipartID = true
```

Comment:
Why do we need to add this in our example?

Reply:
The `multipartId` becomes invalid and there's no need to delete the object when `CompleteMultipart` returns success.

Reply:
We are writing examples here. Our purpose is to show how to use our APIs instead of writing real code.

So please don't introduce too complex logic here.